packages/core/src/components/SupportButton/SupportButton.tsx

```diff
@@ -119,13 +160,12 @@ export const SupportButton = ({ children }: PropsWithChildren<Props>) => {
         onClose={popoverCloseHandler}
       >
         <List className={classes.popoverList}>
-          {React.Children.map(children, (child, i) => (
+          {React.Children.map(title, (child, i) => (
```

Comment:
This seems unnecessary to use React.Children directly on the `title` prop. I also think removing the `children` above will be a breaking change for some plugins that are not in this repo. Maybe something like this could be both backwards compatible:

```tsx
<List className={classes.popoverList}>
  {title && <ListItem alignItems="flex-start">{title}</ListItem>}
  {React.Children.map(children, (child, i) => (
    <ListItem alignItems="flex-start" key={i}>
      {child}
    </ListItem>}
```

Of course this also means going back to using `PropsWithChildren`.